moneta.studio.custom-domain.json

```diff
@@ -0,0 +1,20 @@
+{
+  "providerId": "moneta.studio",
+  "providerName": "Moneta Studio",
+  "serviceId": "custom-domain",
+  "serviceName": "Custom Domain (apex only)",
```

Comment:
I think it may be a bit misleading, as this template in fact does not add RRs to domain APEX, but only to a subdomain.